### ISSUE TITLE
Create a new footer.

### ISSUE THREAD

**Issue by aftonrhodes** [academyforgrowth/WTS_Jumbotron#17]
Make a new footer that allows for the address to be inputted on one line (this will make it easier for production in the var.php).

Use an isolated branch and don't merge until a pull request has been made.

**Comment by aftonrhodes** [comment]
No longer needed.
